src/vmm/src/cpu_config/x86_64/cpuid/normalize.rs

```diff
@@ -247,7 +247,14 @@ impl super::Cpuid {
         // operation using a TSC deadline value.
         //
         // tsc_deadline: 24,
-        set_bit(&mut leaf_1.result.ecx, ECX_TSC_DEADLINE_BITINDEX, true);
+        let host_leaf_1 = cpuid(0x1);
```

Comment:
Lest we forget: in case this makes it off the draft, we'll need to update the doc: https://github.com/firecracker-microvm/firecracker/blob/main/docs/cpu_templates/cpuid-normalization.md